src/promptflow-core/promptflow/core/__init__.py

```diff
@@ -25,6 +25,8 @@
     "tool",
```

Comment:
write await here
![image](https://github.com/microsoft/promptflow/assets/47586720/eca7ad75-e848-40ab-b382-ce53103b5efc)

Reply:
fix it